entity-management-definitions/src/main/java/eu/europeana/entitymanagement/utils/EntityObjectFactory.java

```diff
@@ -84,4 +84,27 @@ public static Entity createEntityObject(String entityType) throws EntityCreation
 						entity.getType(), entity.getEntityId()));
 		}
 	}
+	
+	public static Entity createNewEntity(Entity entity)
+                        throws EntityManagementRuntimeException {
+                // we have to explicitly instantiate the Xml instances, as getDeclaredConstructor().newInstance(args)
```

Comment:
Best to remove this comment as it's confusing 🙂

Reply:
well, this method is needed, eventually we can rename it. This creates the deep clone of the entity